tests/Rector/Contrib/PHPUnit/SpecificMethodRector/SpecificMethodRectorTest.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Rector\Contrib\PHPUnit\SpecificMethodRector;
+
+use Rector\Rector\Contrib\PHPUnit\SpecificMethodRector;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+final class SpecificMethodRectorTest extends AbstractRectorTestCase
+{
+    /**
+     * @dataProvider provideWrongToFixedFiles()
```

Comment:
`dataProvider`s are the best. They will definitely help us simplify our tests!

Reply:
:+1: if you like this, you can send PR for all test to turn like this